### DISCUSSION TITLE
MTV 90's + Club MTV - International channel or not (same video flux)

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#403] [Show and tell]
It seems that MTV 90's is being streamed from one and only version for all ISPs.

Captured recently with my ISP data

![image](https://user-images.githubusercontent.com/30985701/188274815-1a39c3cc-3347-427f-8837-3fafe09c841d.png)

Same for Club MTV (recent channel)

![image](https://user-images.githubusercontent.com/30985701/188274936-32a67814-5fb0-4136-8bee-d94b68f8d2d1.png)

**Comment by benevenstanciano** [comment]
I remember reading MTV was consolidating its networks globally. Having the funny effects like Nickmusic which is programmed in the Netherlands but broadcasting in other countries like Australia, meaning that there's Dutch-language songs on it.

**Reply by BellezaEmporium** [reply]
Which would explain why we have channels in their original form, untouched. I have 4 channels from the MTV group actually (excluding the classic MTV channel), all of them being thematic ones.

**Reply by benevenstanciano** [reply]
I think only the UK market has a different version, as it has commercials while the others don't. Every other version is likely the same.

**Reply by MikeJaibo** [reply]
Do you know if there's a free IPTV link for Club MTV?

**Comment by bsup68** [comment]
Speaking of Club MTV, anyone know which countrys still got the channel?

**Reply by BellezaEmporium** [reply]
France, at least, has it.

**Reply by bsup68** [reply]
Thanks for the info :)

**Reply by BellezaEmporium** [reply]
![image](https://user-images.githubusercontent.com/30985701/236632944-1328322e-372c-4577-8443-841baaaa9fca.png)
Here's all of them.
